gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/ReporterServiceProxyImpl.java

```diff
@@ -94,14 +96,26 @@ public Single<Reporter> create(Reference reference, NewReporter newReporter, Use
 
     @Override
     public Single<Reporter> update(Reference domain, String id, UpdateReporter updateReporter, User principal, boolean isUpgrader) {
+
+        Supplier<ReporterAuditBuilder> baseAudit = () ->
```

Comment:
question: why a supplier, why not a "final var audit = AuditBuilder.builder(... ?